plugins/com.google.cloud.tools.eclipse.ui.util/META-INF/MANIFEST.MF

```diff
@@ -32,6 +32,7 @@ Import-Package: com.google.cloud.tools.eclipse.ui.util,
  org.eclipse.ui.browser;ui.workbench=split,
  org.eclipse.ui.console,
  org.eclipse.ui.handlers,
+ org.eclipse.ui.ide,
```

Comment:
Since we moved this util code, can we remove this from the newproject `MANIFEST.MF`?